### PR TITLE
Change error messages for DeleteCommand

### PR DESCRIPTION
**Pull request by wj331**
1. Change error messages for mainly when deleting -1, 0, 1 and str values to be more specific

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/ParserUtil.java [hunk 1]
```diff
@@ -46,6 +47,28 @@ public static Index parseIndex(String oneBasedIndex) throws ParseException {
...
 
+    /**
+     * Parses {@code oneBasedIndex} into an {@code Index} and returns it. Leading and trailing whitespaces will be
+     * trimmed. This is specifically used for DeleteCommand.
+     * @throws ParseException if the specified index is invalid (less than 1 or not an integer).
+     */
+    public static Index parseDeleteIndex(String oneBasedIndex) throws ParseException {
+        String trimmedIndex = oneBasedIndex.trim();
+        if (trimmedIndex.isEmpty()) {
+            throw new ParseException("");
```
Comment (tohpinren) [inline src/main/java/seedu/address/logic/parser/ParserUtil.java]: Is there a better way to throw this error upwards? Don't think empty error message is good practice. Maybe change to a more relevant message?

**Review by tohpinren — CHANGES_REQUESTED** [review]
Some changes for error message

File: src/main/java/seedu/address/logic/parser/ParserUtil.java (hunk 1 above)
Comment (wj331) [reply src/main/java/seedu/address/logic/parser/ParserUtil.java]: Fixed in latest commit

**Review by tohpinren — APPROVED** [review]

File: src/main/java/seedu/address/logic/parser/ParserUtil.java (hunk 1 above)
Comment (tohpinren) [reply src/main/java/seedu/address/logic/parser/ParserUtil.java]: @wj331 Need to fix checkstyle otherwise lgtm

**Pull request merged**